mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -166,16 +166,21 @@ class SetpointRawPlugin : public plugin::PluginBase,
 			position = ftf::transform_frame_baselink_aircraft(position);
 			velocity = ftf::transform_frame_baselink_aircraft(velocity);
 			af = ftf::transform_frame_baselink_aircraft(af);
+			yaw = ftf::quaternion_get_yaw(
+					ftf::quaternion_from_rpy(M_PI, 0.0, 0.0)*(
+							ftf::quaternion_from_rpy(0.0, 0.0, req->yaw)));
 		} else {
 			position = ftf::transform_frame_enu_ned(position);
 			velocity = ftf::transform_frame_enu_ned(velocity);
 			af = ftf::transform_frame_enu_ned(af);
-		}
-
-		yaw = ftf::quaternion_get_yaw(
+			yaw = ftf::quaternion_get_yaw(
 					ftf::transform_orientation_aircraft_baselink(
 						ftf::transform_orientation_ned_enu(
 							ftf::quaternion_from_rpy(0.0, 0.0, req->yaw))));
+		}
+
+ 		ROS_INFO("FROM MAVROS,YAW=%f,YAW_ORG=%f",ftf::quaternion_get_yaw(
```

Comment:
Please remove the debug message.

Reply:
debug message removed.
the multiplication order is reversed.